app/routes.py

```diff
@@ -20,27 +20,51 @@
     template_folder='../web/dist',
 )
 
+# record user session_id in session
+session = {}
+
 @bp_web_api.route('/ping', methods=['GET'])
 def ping():
     return jsonify('Pong!')
 
-@bp_web_api.route('/data', methods=['GET'])
-def get_data():
+@bp_web_api.route('/auth/login', methods=['POST'])  
+def login():   
+    data = request.get_json()
+
+    # sure data access
+    name = data.get('name')
+    credential = data.get('credential')
```

Comment:
## Warn

Wrong status. These fields are **required**. Should respond with 400.